packages/backend/src/core/activity/processors/createProcessors.ts

```diff
@@ -0,0 +1,180 @@
+import { HttpClient, Logger } from '@l2beat/common'
+import { assert } from '@l2beat/common/src/tools/assert'
+import { Layer2TransactionApi } from '@l2beat/config'
+import { ProjectId } from '@l2beat/types'
+
+import { Config } from '../../../config'
+import { Project } from '../../../model'
+import { BlockTransactionCountRepository } from '../../../peripherals/database/activity-v2/BlockTransactionCountRepository'
+import { StarkexTransactionCountRepository } from '../../../peripherals/database/activity-v2/StarkexCountRepository'
+import { SequenceProcessorRepository } from '../../../peripherals/database/SequenceProcessorRepository'
+import { Database } from '../../../peripherals/database/shared/Database'
+import { StarkexClient } from '../../../peripherals/starkex'
+import { Clock } from '../../Clock'
+import { SequenceProcessor } from '../../SequenceProcessor'
+import { createAztecProcessor } from './createAztecProcessor'
+import { createLoopringProcessor } from './createLoopringProcessor'
+import { createRpcProcessor } from './createRpcProcessor'
+import { createStarkexProcessor } from './createStarkexProcessor'
+import { createStarknetProcessor } from './createStarknetProcessor'
+import { createZksyncProcessor } from './createZksyncProcessor'
+
+export function createSequenceProcessors(
```

Comment:
This file should probably be moved to setup near ActivityV2Module and renamed to `createSequenceProcessors`